### DISCUSSION TITLE
vqgan-clip-generator

### DISCUSSION THREAD

**Discussion by PhotoCanvas** [rkhamilton/vqgan-clip-generator#70] [General]
Hi all..

i d like to make some very basic videos startin from prompt txt..  i m still in noob mode, learning..

so, i have an issue with starting test 

git clone git@github.com:hzwer/arXiv2020-RIFE.git
Cloning into 'arXiv2020-RIFE'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

so does repository exists or it was deleted?

thank you for all info or other beginners guides regarding "making ai video" topics:)

Blaz

**Comment by rkhamilton** [comment]
The RIFE2020 repository is [here](https://github.com/hzwer/arXiv2020-RIFE) and I am not affiliated with them. If you are having difficulty installing their package then they would be a better group to ask.

The specific error you are getting sounds like a problem with your SSH key. Check [this link](https://stackoverflow.com/questions/57734669/gitgithub-com-permission-denied-publickey) for discussion of that error and how some people have resolved it.
